docs/source/about/guides/linux.rst

```diff
@@ -0,0 +1,459 @@
+#. Optionally setup a ``~/.ssh/config`` file to simplify the ``ssh`` command
+   
+   .. code-block::
+
+		Host <some_alias>
+		    Hostname <ip_address>
+		    User <username>
+		    IdentityFile ~/.ssh/<your_private_key>
+
+   Now you can use ``ssh <some_alias>``.  
+   ``ssh <some_alias> <commands/script>`` will execute the command or script on the remote host.
+
+Checkpoint
+++++++++++
+
+Let's make sure your setup is working so far!
+
+**Test Steps**
+
+With your monitor still plugged into your PC:
+
+#. ``ssh <alias>``
+#. ``~/scripts/sunshine.sh``
+#. ``nvidia-smi``
+
+   You should see the sunshine and Xorg processing running:
+
+   .. code-block::
+       
+       $ nvidia-smi
+       Tue Aug 29 18:38:46 2023
+       +---------------------------------------------------------------------------------------+
+       | NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2     |
+       |-----------------------------------------+----------------------+----------------------+
+       | GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
+       | Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
+       |                                         |                      |               MIG M. |
+       |=========================================+======================+======================|
+       |   0  NVIDIA GeForce RTX 3070        Off | 00000000:01:00.0  On |                  N/A |
+       | 30%   46C    P2              45W / 220W |    549MiB /  8192MiB |      2%      Default |
+       |                                         |                      |                  N/A |
+       +-----------------------------------------+----------------------+----------------------+
+       
+       +---------------------------------------------------------------------------------------+
+       | Processes:                                                                            |
+       |  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
+       |        ID   ID                                                             Usage      |
+       |=======================================================================================|
+       |    0   N/A  N/A      1393      G   /usr/lib/Xorg                                86MiB |
+       |    0   N/A  N/A      1440    C+G   sunshine                                    293MiB |
+       +---------------------------------------------------------------------------------------+
+ 
+#. Check ``/dev/uinput`` permissions
+   
+   .. code-block:: bash
+
+       $ ls -l /dev/uinput
+       crw------- 1 <user> <primary_group> 10, 223 Aug 29 17:31 /dev/uinput
+
+#. Connect to host from a moonlight client
+
+*Now unplug your monitors and repeat steps 1 - 5*
```

Comment:
Clarify what different result is expected?

Reply:
The result should be the same. Being able to successfully  start the X server without monitors plugged in and having sunshine working as expected.